teuthology/task/__init__.py

```diff
@@ -13,9 +13,11 @@ class Task(object):
     Can be used as a drop-in replacement for the old-style task functions with
     @contextmanager decorators.
     """
+
+    name = 'task'
```

Comment:
Do all class-based tasks require this 'name' property now?  If so, maybe some docs about this are necessary?